go/vt/vtgate/engine/route.go

```diff
@@ -278,7 +278,9 @@ func (route *Route) StreamExecute(vcursor VCursor, bindVars map[string]*querypb.
 		defer cancel()
 	}
 	switch route.Opcode {
-	case SelectUnsharded, SelectScatter:
+	case SelectUnsharded, SelectNext, SelectDBA, SelectReference:
```

Comment:
These are essentially unreachable code paths or impractical use cases. But I decided to add these for the sake of correctness and completeness. Otherwise, it becomes hard to reason about why these got left out.